pyproject.toml

```diff
@@ -22,6 +22,11 @@ pytest-cov = "^4.1.0"
 numpy = "^1.24.2"
 pytest-benchmark = "^4.0.0"
 
+[tool.poetry.group.test.dependencies]
+matplotlib = "^3.7.0"
+orion = "^0.1.0"  
+kaleido = "0.2.1" 
```

Comment:
Why a dev and a test group? Can't we put those in the dev group as well?

Reply:
Yes we can 😄